test/regress/regress0/arith/arith-eq.smt2

```diff
@@ -0,0 +1,15 @@
+; COMMAND-LINE: --no-check-models  --no-check-unsat-cores --no-check-proofs
```

Comment:
Is `--no-check-models` necessary?

Reply:
In haiku:
copy-paste strikes again

Reply:
:)
In that case, I also don't find the smtlib version that important either (I am assuming it is there for the same reason).

Reply:
BTW you have 6 syllables in the third line...